src/lib.rs

```diff
@@ -4525,6 +4528,48 @@ pub unsafe trait FromBytes: FromZeros {
         }
     }
 
+    /// Reads a copy of `self` from the IO byte source `src`.
+    ///
+    /// This is useful for interfacing with operating system byte sinks (files,
+    /// sockets, etc.).
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
```

Comment:
If you want this to run, you could use a trick like we do here: https://github.com/google/zerocopy/blob/ace5bb0de76e150d2284984677892a5aaa34a36e/src/macros.rs#L711